client/styles/debug.css

```diff
@@ -71,14 +82,25 @@ a:active {
     border-bottom: solid 1px #004e7f;
     color: white;
     background: #005a93;
+    padding: 15px;
+}
+
+.header.info.error {
+    margin: 15px -15px 15px -33px;
```

Comment:
I think these margins will need tweaking, see the header in the screenshot above